test/e2e-common.sh

```diff
@@ -43,7 +42,7 @@ function install_pipeline_crd() {
   if [[ -n ${RELEASE_YAML} ]];then
 	latestreleaseyaml=${RELEASE_YAML}
   else
-    latestreleaseyaml=$(curl -s https://api.github.com/repos/tektoncd/pipeline/releases|python -c "import sys, json;x=json.load(sys.stdin);ass=x[0]['assets'];print([ x['browser_download_url'] for x in ass if x['name'] == 'release.yaml'][0])")
```

Comment:
Could this just default to https://storage.googleapis.com/tekton-releases/latest/release.yaml, instead of querying GitHub for the latest release?

Reply:
:+1: nice catch